com.unity.render-pipelines.universal/Documentation~/writing-shaders-urp-basic-unlit-structure.md

```diff
@@ -132,17 +131,46 @@ Tags { "RenderType" = "Opaque" "RenderPipeline" = "UniversalPipeline" }
 
 A SubShader Tag with a name of `RenderPipeline` tells Unity which render pipelines to use this SubShader with, and the value of `UniversalPipeline` indicates that Unity should use this SubShader with URP.
 
+You can create multiple SubShaders with different `RenderPipeline` tags if you wish to execute the same shader in different render pipelines.
```

Comment:
Since there are only two other pipelines, I guess we could just add the two values here. What are the values for Built-in and HDRP?